packages/e2e-test-app/package.json

```diff
@@ -41,8 +41,6 @@
     "babel-jest": "^26.3.0",
     "eslint": "^7.32.0",
     "jest": "^26.6.3",
-    "just-scripts": "^1.3.3",
-    "metro-config": "^0.67.0",
```

Comment:
We now dictate the version of `metro-config` in `@rnw-scripts/metro-dev-config`.